test/hotspot/jtreg/compiler/jvmci/jdk.vm.ci.runtime.test/src/jdk/vm/ci/runtime/test/TestResolvedJavaField.java

```diff
@@ -93,6 +93,16 @@ public void getModifiersTest() {
         }
     }
 
+    @Test
+    public void getInternalModifiersTest() {
```

Comment:
No need for this test since the `getInternalModifiers` method should be removed.